src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/WorkflowComponent.scala

```diff
@@ -280,29 +251,12 @@ trait WorkflowComponent {
         findWorkflowById(id).delete
     }
 
-    def deleteWorkflowErrors(submissionId: UUID) = {
-      findInactiveWorkflows(submissionId).result flatMap { result =>
-        DBIO.seq(result.map(f => workflowQuery.findWorkflowErrorsByWorkflowFailureId(f.id).delete).toSeq:_*)
-      }
-    }
-
-    def deleteWorkflowFailures(submissionId: UUID) = {
-      submissionQuery.filter(_.id === submissionId).result flatMap { result =>
-        DBIO.seq(result.map(sub => workflowQuery.findInactiveWorkflows(submissionId).delete).toSeq:_*)
-      }
-    }
-
     //gather all workflowIds and workflowFailureIds in a submission and delete their attributes
     def deleteSubmissionAttributes(submissionId: UUID) = {
       val workflows = workflowQuery.filter(_.submissionId === submissionId).result
-      val workflowFailures = workflowFailureQuery.filter(_.submissionId === submissionId).result
 
-      workflows flatMap { workflowRecs =>
+      workflows map { workflowRecs =>
         val workflowIds = workflowRecs.map(_.id)
```

Comment:
I don't think you want "val workflowIds =" here. It makes the overall result Unit I think.